index.js

```diff
@@ -55,72 +61,86 @@ async function eslintBankruptcy(options) {
  * @param {string | undefined} explanation
  */
 function insertComments(changes, explanation) {
   // @codemod/cli has more functionality, but it'll be painful to use because we'd have to run it in subproc.
   // Our set of changes to make is in memory, so passing that through to the transform would also be a pain.
 
-  return Promise.all(_.map(changes, (violations, filePath) => insertCommentsInFile(filePath, violations, explanation)));
+  return Promise.all(
+    _.map(changes, (violations, filePath) =>
+      insertCommentsInFile(filePath, violations, explanation)
+    )
+  );
 }
 
 /**
- * @param {string} filePath 
- * @param {{[line: number]: string[]}} violations 
- * @param {string | undefined} explanation 
+ * @param {string} filePath
+ * @param {{[line: number]: string[]}} violations
+ * @param {string | undefined} explanation
  */
 async function insertCommentsInFile(filePath, violations, explanation) {
   log.info({filePath}, 'Modifying file');
   // I wonder if the line splitting is too naive here.
   const inputCode = (await readFile(filePath, 'utf8')).split('\n');
-  
+
   // I would declare this inline if I knew how to use the TS JSDoc syntax with it.
   /** @type {string[]} */
   const initial = [];
-  
-  const outputCode = inputCode.reduce((acc, line, lineIndex) => {
-    const toAppend = [];
-    // +1 because ESLint gives the line numbers 1-indexed.
-    const violation = violations[lineIndex + 1];
-    if (violation) {
-      const leadingWhitespaceLength = line.length - line.trimLeft().length;
-      const leadingWhitespace = line.substring(0, leadingWhitespaceLength);
-      if (explanation) {
-        toAppend.push(`${leadingWhitespace}// ${explanation}`);
+
+  const outputCode = inputCode
+    .reduce((acc, line, lineIndex) => {
+      const toAppend = [];
+      // +1 because ESLint gives the line numbers 1-indexed.
+      const violation = violations[lineIndex + 1];
+      if (violation) {
+        const leadingWhitespaceLength = line.length - line.trimLeft().length;
+        const leadingWhitespace = line.substring(0, leadingWhitespaceLength);
+        if (explanation) {
+          toAppend.push(`${leadingWhitespace}/* ${explanation} */`);
+        }
+        toAppend.push(leadingWhitespace + getEslintDisableComment(violation));
       }
-      toAppend.push(leadingWhitespace + getEslintDisableComent(violation));
-    }
-    toAppend.push(line);
-    return [...acc, ...toAppend];
-  }, initial).join('\n');
+      toAppend.push(line);
+      return [...acc, ...toAppend];
+    }, initial)
+    .join('\n');
 
   log.trace({outputCode, filePath});
   await writeFile(filePath, outputCode);
 }
 
 /**
- * @param {string[]} rules 
+ * @param {string[]} rules
  */
-function getEslintDisableComent(rules) {
-  return `// eslint-disable-next-line ${rules.join(' ')}`;
+function getEslintDisableComment(rules) {
+  const ruleSet = new Set(rules);
+  return `/* eslint-disable-next-line ${Array.from(ruleSet).join(', ')} */`;
```

Comment:
using `/* ... */` style comments made it easier to use regex to look for the new comments and fix broken comments in jsx